src/current/v23.2/create-virtual-cluster.md

```diff
@@ -0,0 +1,110 @@
+</div>
+
+## Parameters
+
+Parameter | Description
+----------+------------
+`virtual_cluster_name` | The name for the new virtual cluster.
+`LIKE virtual_cluster_spec` | Ensures the virtual cluster on the standby is created with the correct [capabilities](#capabilities).
+`primary_virtual_cluster` | The name of the primary's virtual cluster to replicate.
+`primary_connection_string` | The PostgreSQL connection string to the primary cluster. Refer to [Connection string](#connection-string) for more detail.
+`replication_options_list`| Options to modify the replication streams. Refer to [Options](#options).
+
+## Options
+
+Option | Description
+-------+-------------
+`RETENTION` | Configure a [retention window]({% link {{ page.version.version }}/physical-cluster-replication-technical-overview.md %}#cutover-and-promotion-process) that will control how far in the past you can [cut over]({% link {{ page.version.version }}/cutover-replication.md %}) to.
+
+## Connection string
+
+When you [initiate a replication stream]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#step-4-start-replication) from the standby cluster, it is necessary to pass a connection string to the primary cluster:
+
+{% include_cached copy-clipboard.html %}
+~~~
+'postgresql://{replication user}:{password}@{node IP or hostname}:26257/?options=-ccluster=system&sslmode=verify-full&sslrootcert=certs/{primary cert}.crt'
+~~~
+
+To form the connection string to the primary cluster, use the following query parameters:
+
+Query parameter | Description
+----------------+------------
+`replication user` | The user on the primary cluster that has the `REPLICATION` system privilege. Refer to the [Create a replication user and password]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#create-a-replication-user-and-password) for more detail.
+`password` | The replication user's password.
+`node ID or hostname` | The node IP address or hostname of any node from the primary cluster.
+`ccluster=system` | The parameter to connect to the system interface on the primary cluster.
+`primary cert` | The path to the primary cluster's certificate on the standby cluster.
+
+## Capabilities
+
+{{site.data.alerts.callout_info}}
+Cockroach Labs does not recommend changing the default capabilities of created virtual clusters.
+{{site.data.alerts.end}}
+
+_Capabilities_ control what a virtual cluster can do. The [configuration profile]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#start-the-standby-cluster) included at startup creates the `template` virtual cluster with the same set of capabilities per CockroachDB version. When you start a replication stream, you can specify the `template` VC with `LIKE` to ensure other virtual clusters on the standby cluster will work in the same way. `LIKE` will refer to a virtual cluster on the CockroachDB cluster you're running the statement from.
+
```

Comment:
:D I know I made a big deal about including this. But, I then realized that as part of the replication profile, the `template` will actually be the default. So if they follow our instructions, they shouldn't need to specify the `LIKE` clause anymore.

But, I think it is fine to leave it as is.

Reply:
@stevendanna OK! Will this be more useful when customers might be using virtual clusters other than just for PCR? I don't want to leave it in, if it's confusing... but perhaps it won't hurt anyway

Reply:
I was wondering if a customer will use `LIKE` beyond `template` right now. If that's an OK usage then it probably serves to leave this in.